data/txHandler.go

```diff
@@ -156,17 +165,55 @@ func (handler *TxHandler) backlogWorker() {
 			if !ok {
 				return
 			}
-			handler.postprocessCheckedTxn(wi)
+			handler.postProcessCheckedTxn(wi)
 
 		case <-handler.ctx.Done():
 			return
 		}
 	}
 }
 
-func (handler *TxHandler) postprocessCheckedTxn(wi *txBacklogMsg) {
+func (handler *TxHandler) postProcessReportErrors(err error) {
```

Comment:
agree, looks `var feeError *verify.ErrTxGroupInvalidFee` and to escape to heap.
The problem is with the "reason" enum is 1) you still need to somehow convert errors to reasons 2) errors.Is/As are made for this.
So the solution might be making these declarations globals to prevent allocations, and this looks safe since `postProcessCheckedTxn` is single threaded

Reply:
Added ErrTxGroupError with the reason code, so only one allocation